protocol/x/accountplus/keeper/keeper.go

```diff
@@ -62,6 +63,40 @@ func (k Keeper) SetGenesisState(ctx sdk.Context, data types.GenesisState) error
 	return nil
 }
 
+func (k Keeper) InitializeAccount(ctx sdk.Context, address sdk.AccAddress) (types.AccountState, error) {
```

Comment:
This function is not used yet right? When will it be used? Trying to understand why we need this function at all (initializing to empty account state)

Reply:
Sorry I actually wanted to remove this function for this CR but the commit history got kinda messed up. An upcoming PR will implement business logic in the validator and will use this function. Feel free to review this then.

Reply:
When would we want to initialize empty account state? Wouldn't it always be non-empty when it's first created (i.e. one nonce?)

Reply:
My thought for initializing account was that as users submit tx, if they did not previously have an AccountState (corresponding to accountplus), a new account with default substructs (eg. timestamp nonce {[], 0} will be created. 

Each subcomponent of accountplusmodule, such as timestamp nonce, will be responsible for updating the default state when it becomes relevant. In ts-nonce case, it will be when the user first uses a ts-nonce.

Reply:
> My thought for initializing account was that as users submit tx, if they did not previously have an AccountState (corresponding to accountplus), a new account with default substructs (eg. timestamp nonce {[], 0} will be created.

This seems pretty wasteful. In general state storage is a scare resource in blockchain, so we prefer to not add a node vs. having an empty node. Similarly, there was a project to remove empty subaccount state when their no longer have any outstanding balance/positions.

Also it seems natural that any transaction that's no using this special timestamp nonce should no result in any state change in `x/accountplus` state